ctpolicy/ctpolicy.go

```diff
@@ -90,7 +90,7 @@ func (ctp *CTPolicy) race(ctx context.Context, cert core.CertDER, group cmd.CTGr
 
 // GetSCTs attempts to retrieve a SCT from each configured grouping of logs and returns
 // the set of SCTs to the caller.
-func (ctp *CTPolicy) GetSCTs(ctx context.Context, cert core.CertDER) ([]core.SCTDER, error) {
+func (ctp *CTPolicy) GetSCTs(ctx context.Context, cert core.CertDER) ([][]byte, error) {
```

Comment:
Why did this type change back?

Reply:
Ah, I meant to add a comment about this. The protobuf messages we generate translate `repeated bytes` to `[][]byte` and golang won't let you use a `[]typedBytes` (where `type typedBytes []byte`) for that. We could add a translation in the gRPC wrappers on each side of the RPC that took a `[]core.SCTDER` and turned it into a `[][]byte` and vice-versa but that seems rather memory inefficient just to gain a named type.

Reply:
What about creating a new named type, `SCTDERs` that maps to `[][]byte`? Then we could just cast to and from that.